integration-tests/smoke/auto_ocr_test.go

```diff
@@ -0,0 +1,181 @@
+	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
+	"github.com/smartcontractkit/chainlink-testing-framework/contracts/ethereum"
+	"github.com/smartcontractkit/chainlink-testing-framework/utils"
+
+	networks "github.com/smartcontractkit/chainlink/integration-tests"
+	"github.com/smartcontractkit/chainlink/integration-tests/actions"
+	"github.com/smartcontractkit/chainlink/integration-tests/client"
+	"github.com/smartcontractkit/chainlink/integration-tests/contracts"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/rs/zerolog/log"
+)
+
+var _ = Describe("Automation OCR Suite @keeper", func() {
+	var (
+		err              error
+		chainClient      blockchain.EVMClient
+		contractDeployer contracts.ContractDeployer
+		registry         contracts.KeeperRegistry
+		registrar        contracts.KeeperRegistrar
+		consumers        []contracts.KeeperConsumer
+		upkeepIDs        []*big.Int
+		linkToken        contracts.LinkToken
+		chainlinkNodes   []*client.Chainlink
+		testEnvironment  *environment.Environment
+
+		testScenarios = []TableEntry{
+			Entry("v2.0 Basic smoke test @simulated", ethereum.RegistryVersion_2_0, defaultRegistryConfig, BasicCounter, BasicSmokeTest, big.NewInt(defaultLinkFunds), 2),
+		}
+	)
+
+	DescribeTable("Automation OCR Suite @keeper", func(
+		registryVersion ethereum.KeeperRegistryVersion,
+		registryConfig contracts.KeeperRegistrySettings,
+		consumerContract KeeperConsumerContracts,
+		testToRun KeeperTests,
+		linkFundsForEachUpkeep *big.Int,
+		numberOfUpkeeps int,
+	) {
+		By("Deploying the environment")
+		envVars := map[string]interface{}{
+			"MIN_INCOMING_CONFIRMATIONS":  "1",
+			"KEEPER_TURN_LOOK_BACK":       "0",
+			"FEATURE_OFFCHAIN_REPORTING2": "true",
+			"FEATURE_LOG_POLLER":          "true",
+			"P2P_NETWORKING_STACK":        "V2",
+			"P2P_LISTEN_PORT":             "",
+			"CHAINLINK_TLS_PORT":          "0",
+			"P2PV2_LISTEN_ADDRESSES":      "0.0.0.0:6690",
+			"P2PV2_ANNOUNCE_ADDRESSES":    "0.0.0.0:6690",
+		}
+		testEnvironment = environment.New(&environment.Config{NamespacePrefix: "smoke-keeper"}).
+			AddHelm(mockservercfg.New(nil)).
+			AddHelm(mockserver.New(nil)).
+			AddHelm(eth.New(nil)).
+			AddHelm(chainlink.New(0, map[string]interface{}{
+				"replicas": "5",
+				"env":      envVars,
+			}))
+		err = testEnvironment.Run()
+		Expect(err).ShouldNot(HaveOccurred())
+
+		By("Connecting to launched resources")
+		network := networks.SimulatedEVM
+		chainClient, err = blockchain.NewEVMClient(network, testEnvironment)
+		Expect(err).ShouldNot(HaveOccurred(), "Connecting to blockchain nodes shouldn't fail")
+		contractDeployer, err = contracts.NewContractDeployer(chainClient)
+		Expect(err).ShouldNot(HaveOccurred(), "Deploying contracts shouldn't fail")
+		chainlinkNodes, err = client.ConnectChainlinkNodes(testEnvironment)
+		Expect(err).ShouldNot(HaveOccurred(), "Connecting to chainlink nodes shouldn't fail")
```

Comment:
Nit: You can call [this](https://github.com/smartcontractkit/chainlink/blob/acdbe197438553b2c5effdf6b4f5ab6d8a0f09e8/integration-tests/client/chainlink.go#L998) somewhere here to avoid subsequent api calls to read keys.

Reply:
addressed the rest but this one caused errors. So i will look at it in my next PR when I add more tests but would like to get this first basic smoke test in